experiments/distributedshell/src/main/scala/org/apache/gearpump/distributedshell/DistributedShell.scala

```diff
@@ -31,7 +31,7 @@ object DistributedShell extends App with ArgumentsParser {
   )
 
   def application(config: ParseResult) : Application = {
-    val app = Application("DistributedShell", classOf[AppMaster].getCanonicalName, UserConfig.empty)
+    val app = Application("DistributedShell", classOf[DistShellAppMaster].getCanonicalName, UserConfig.empty)
```

Comment:
getName instead of getCanonicalName

getCanonicalName treate differently with getName when the class is in companion object.